### ISSUE TITLE
Exclude front matter from reading time calculation

### ISSUE THREAD

**Issue by redactedscribe** [avr/obsidian-reading-time#8]
The current reading time value increases the longer one's front matter is. Front matter should be excluded from how the plugin calculates the word count.

Thanks.

**Comment by avr** [comment]
Thanks - good suggestion. I'll try to take a look. (This may be similar to #6 - which I haven't explored).)

**Comment by redactedscribe** [comment]
Yes, embeds should probably be added towards the word count of the document by default, however I think it would be convenient to be able to see the reading time both with and without embeds included, and for the entire document (which is the currently displayed value pre any fix).

For example:

- An option to choose which of the following values is displayed in the status bar.
- When hovering over the reading time in the status bar, or clicking it, reveal several reading time values:
  - **Reading time: 3 minutes**
    - I.e. (body + embeds) - front matter.
  - **Excluding embeds: 2 minutes**
    - Body - (embeds - front matter).
  - **All content: 4 minutes**
    - All words in the document.
